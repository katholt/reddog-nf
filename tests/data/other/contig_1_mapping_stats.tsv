isolate	replicon_coverage	replicon_average_depth	replicon_reads_mapped	total_mapped_reads	total_reads	snps	snps_heterozygous	indels	pass_fail	phylogeny_group
isolate_17	99.94	10.65	51.372	35.0053	3748	0	0	0	f	n/a
isolate_14	99.94	9.57	51.3559	100.0	1180	0	0	0	f	n/a
isolate_12	99.94	14.5	51.3998	35.0059	5102	0	0	0	f	n/a
isolate_13	99.94	8.72	51.3966	100.0	1074	0	0	0	f	n/a
isolate_15	99.94	8.72	51.3966	55.0205	1952	0	0	0	f	n/a
isolate_16	99.94	8.72	51.3966	35.0065	3068	0	0	0	f	n/a
isolate_11	99.94	14.5	51.3998	20.0	8930	0	0	0	f	n/a
isolate_6	99.94	14.5	51.3998	50.0	3572	0	0	0	p	i
isolate_9	99.94	23.88	51.3936	100.0	2942	0	0	0	p	i
isolate_7	99.94	14.5	51.3998	54.9877	3248	0	0	0	p	i
isolate_23	99.94	14.5	51.3998	100.0	1786	1	0	0	p	i
isolate_1	99.94	14.49	51.3998	100.0	1786	19	0	0	p	o
isolate_2	99.94	14.49	51.3998	100.0	1786	5	0	0	p	i
isolate_18	99.94	14.5	51.3998	100.0	1786	2	0	0	p	i
isolate_22	99.94	14.5	51.3998	100.0	1786	1	0	0	p	i
isolate_37	99.94	14.5	51.3998	100.0	1786	1	0	0	p	i
isolate_32	99.94	14.5	51.3998	100.0	1786	1	0	0	p	i
isolate_20	99.94	14.5	51.3998	100.0	1786	2	0	0	p	i
isolate_4	99.94	14.5	51.3998	100.0	1786	0	0	1	p	i
isolate_3	99.94	14.5	51.3998	100.0	1786	0	3	0	p	i
isolate_10	99.94	10.65	51.372	54.9874	2386	0	0	0	p	i
isolate_30	99.94	14.5	51.3998	100.0	1786	1	0	0	p	i
isolate_25	99.94	14.5	51.3998	100.0	1786	1	0	0	p	i
isolate_29	99.94	14.5	51.3998	100.0	1786	1	0	0	p	i
isolate_8	99.94	10.65	51.372	100.0	1312	0	0	0	p	i
isolate_21	99.94	14.5	51.3998	100.0	1786	1	0	0	p	i
isolate_28	99.94	14.5	51.3998	100.0	1786	1	0	0	p	i
isolate_27	99.94	14.5	51.3998	100.0	1786	1	0	0	p	i
isolate_19	99.94	14.5	51.3998	100.0	1786	1	0	0	p	i
isolate_24	99.94	14.5	51.3998	100.0	1786	1	0	0	p	i
isolate_5	99.94	14.5	51.3998	100.0	1786	0	0	1	p	i
isolate_36	99.94	14.5	51.3998	100.0	1786	1	0	0	p	i
isolate_33	99.94	14.5	51.3998	100.0	1786	1	0	0	p	i
isolate_26	99.94	14.5	51.3998	100.0	1786	1	0	0	p	i
isolate_35	99.94	14.5	51.3998	100.0	1786	1	0	0	p	i
isolate_34	99.94	14.5	51.3998	100.0	1786	1	0	0	p	i
isolate_31	99.94	14.5	51.3998	100.0	1786	1	0	0	p	i
