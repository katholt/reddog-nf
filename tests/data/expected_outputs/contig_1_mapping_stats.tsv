../other/contig_1_mapping_stats.tsv